Supporting sidecar view directory (#335)
github/view_component: README.md

end
```

### Sidecar Assets

ViewComponents supports two options for defining view files. 

#### Sidecar view

The simplest option is to place the view next to the Ruby component:

```
app/components
├── ...
├── test_component.rb
├── test_component.html.erb
├── ...
```

#### Sidecar directory

> Alternatively views and other assets can be placed in a sidecar directory with the same name as the component.
> This configuration is particulary useful to organize views along side other assets like javascript and css.

Reviewer: ```suggestion
As an alternative, views and other assets can be placed in a sidecar directory with the same name as the component, which can be useful for organizing views alongside other assets like Javascript and CSS.
```
